emperor/support_files/js/color-editor.js

```diff
@@ -9,21 +14,21 @@ function($, _, DecompositionView, ViewControllers, spectrum) {
 
   /**
    *
-   * @name ColorEditor
+   * @class ColorEditor
    *
-   * @class This class represents a color editor defined by the SlickGrid
+   * This class represents a color editor defined by the SlickGrid
    * project.
    *
-   * @property {$input} object where the Spectrum color picker lives.
-   * @property {defaultValue} initial value of the cell being edited.
-   *
    * Note, this object is heavily based on classes in slick.editors.js and in
-   * the documentation that can be found here:
-   *    https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors
+   * the documentation that can be found [here](https://github.com/mleibman/
```

Comment:
To be more "JSDoc-y", you can change
(https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors)
 to
{@link https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors}
and it will still render as a link.